src/pages/patientView/patientHeader/mock/gdd/P-0001055-T01-IM3

```diff
@@ -0,0 +1,38 @@
+{
```

Comment:
example file of output of one patient. Eventually the data will just be stored in clinical attributes, so this is a temporary thing.